### DISCUSSION TITLE
Add some "resistance" when snapping between two monitors

### DISCUSSION THREAD

**Discussion by chuakc92** [rxhanson/Rectangle#861] [Ideas]
In the windows implementation, if you have two monitors side by side and if you're trying to snap to the area between the two monitors, dragging a window to the side provides a little bit of "resistance" to prevent you from accidentally dragging the window to the next monitor. I'm surprised there is no implementation of this useful feature, or at least an option to enable it. Can this be considered at all?

**Comment by rxhanson** [comment]
This would be a good addition to the app. I don't have the bandwidth to add it at the moment, but I welcome a good pull request for it.
